commands.txt

```diff
@@ -0,0 +1,7 @@
+available commands:
+1. start the service with `confluent local kafka start`
```

Comment:
Any future plans to start all of the services with `confluent local start`? (once we have more than one)